Add some missing PWN 3.1 ILI maps

jmccrae opened the pull request:
As discussed in #16 some of the mappings that are reported as changed in PWN 3.1 are in fact not and have ILI codes. This PR fixes this by putting back all the matching synsets with the correct ILI

jmccrae committed:
Add some missing PWN 3.1 ILI maps

jmccrae committed:
'annulet' and 'roundel' were split in PWN3.1 so need a new synset

jmccrae committed:
Also remove dietician, recycling and decadent which were split

fcbond commented:
This has several mappings from one ili to multiple synsets, which is not allowed:

- we should probably try to add an automated test for this
- a simple one-liner for the tab file is
```
$ sort ili-map-pwn31.tab | cut -f 1 | uniq -d
i10616
i114673
i114674
i114676
i114677
i114679
i114680
i16842
i25156
i34688
i3764
i4234
i4304
i55794
i55822
i56320
i56630
```

I think we should also check the `ili.ttl` file as well.

jmccrae committed:
Update mapping to PWN 3.1

jmccrae committed:
Update change list

Remove all deprecations as these are not clear and some have been
restored by other projects

jmccrae commented:
I updated the branch so it has no duplicates and ILI to PWN 3.1 mappings are correct
